eslint/babel-eslint-parser/package.json

```diff
@@ -15,12 +14,17 @@
   },
   "homepage": "https://babeljs.io/",
   "engines": {
-    "node": ">=10.9"
+    "node": "^10.13.0 || ^12.13.0 || >=14.0.0"
```

Comment:
I feel like we can enforce the same ranges for Babel 8: that we don't have to fight against early node v13 versions that may complain about incorrect `exports` usage.